### PR TITLE
use FileFilter in directory update

### PR DESCRIPTION
**Pull request by tisoft**

### CODE REVIEW

File: src/main/java/com/ricardojlrufino/jexplorer/FileTreeModel.java [hunk 1]
```diff
@@ -18,10 +21,17 @@
 
 public class FileTreeModel extends DefaultTreeModel {
 
-  public FileTreeModel(TreeNode root) {
```
Comment (ricardojlrufino) [inline src/main/java/com/ricardojlrufino/jexplorer/FileTreeModel.java:21]: Please keep this constructor

**Review by ricardojlrufino — COMMENTED** [review]
Hi, Only one small change to merge...

If you use in other "public" project please update README (section: Projects Using)

**Comment by tisoft** [comment]
I have rewritten the whole file filter handling and added a filter textbox to the FileExplorerPanelDemo

I don't have a public project right now, that is using it.

**Review by ricardojlrufino — APPROVED** [review]
Merged... Thank you for contributing

**Pull request merged**